recipes/xmlsec/all/conanfile.py

```diff
@@ -37,7 +37,21 @@ def _source_subfolder(self):
 
     @property
     def _is_msvc(self):
-        return self.settings.compiler == "Visual Studio"
+        return str(self.settings.compiler) in ["Visual Studio", "msvc"]
+
+    @property
+    def _vc_runtime(self):
```

Comment:
I was told there is a helper for that: `msvc_runtime_flag`, but it doesn't seem to be documented 👁️

Reply:
Indeed, since 1.35.0: https://github.com/conan-io/conan/blob/release/1.35/conan/tools/microsoft/visual.py#L7-L18

But it's less robust than this custom method when consumer doesn't have msvc in its `settings.yml`.

Reply:
:facepalm:

Reply:
PR for docs: https://github.com/conan-io/docs/pull/2363